mysql-test/t/audit_log_many_connections.test

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
Since this is not an upstream bug anymore, the conversion to audit_null becomes only "nice to have", up to you to decide. Otherwise LGTM